src/AvatarList/index.js

```diff
@@ -33,19 +33,28 @@ class AvatarList extends Component {
   }
 
   render() {
-    const { imageList, imageSpacing, imageChild, _fonts, listEmptyState } =
-      this.props
+    const {
+      imageList,
+      imageSpacing,
+      imageChild,
+      _fonts,
+      listEmptyState,
+      openAccordion,
+    } = this.props
 
     if (!imageList || typeof navigator.userAgent === undefined) {
       return <View style={{ height: imageSize }}></View>
     }
 
-    if (!imageList[0]) {
+    const renderEmptyState =
+      (imageList && !imageList[0]) || openAccordion === 'listEmptyState'
+
+    if (renderEmptyState) {
       return (
-        <EmptyListWrapper
+        <EmptyState
           listEmptyState={listEmptyState}
           items={imageList}
```

Comment:
```suggestion
```
If you are keeping the render logic inside the individual components, you shouldn't need to pass in `items` anymore. 
Additionally, you could actually just spread `listEmptyState` to make it even simpler: `<EmptyState {...listEmptyState}/>`